research/delf/delf/python/training/download_dataset.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+# This script dowloands the Google Landmarks v2 dataset. GLDv2 datase image files by split:
+# - TRAIN: 500
+# - INDEX: 100
+# - TEST: 20
```

Comment:
would you mind adding some comment here to explain exactly how this script should be run? Something like:

"Run this script by

sh download_dataset.sh 500 100 20

"

or something like this?

Reply:
And similarly for the install_delf.sh script :)

Reply:
Sure, that's a good ides:) Done for both.